test/AdoNetCore.AseClient.Tests/Unit/LoginPacketTests.cs

```diff
@@ -9,7 +9,7 @@
 
 namespace AdoNetCore.AseClient.Tests.Unit
 {
-    [TestFixture]
+    [TestFixture(Explicit = true, Reason = "The capability bits of this data provider have diverged from the SAP provider.")]
```

Comment:
If that's the case, may as well delete this test file

Reply:
Done.